src/utilities/data/Vector.cpp

```diff
@@ -130,6 +146,10 @@ double interp(const Vector& x, const Vector& y, double xi, InterpMethod interpMe
         // linear interpolation
         result = info.wa * y(info.ia) + info.wb * y(info.ib);
         break;
+      case AverageInterp:
+        // average interpolation
+        result = (info.wa * y(info.ia) + info.wb * y(info.ib)) / info.ti;
+        break;
```

Comment:
Reading this makes me thing that the default ti = 0.0 is weird, shouldn't it be 1.0?


**Edit**: given that do you equality check == 0.0 in interpInfo I don't think it should be either